packages/components/src/ActionIntercom/Intercom.component.js

```diff
@@ -38,6 +38,7 @@ function Intercom({ id, className, config, t, tooltipPlacement }) {
 	return (
 		<TooltipTrigger label={label} tooltipPlacement={tooltipPlacement}>
 			<button
+				data-feature={show ? 'close-ipc' : 'open-ipc'}
```

Comment:
the format used by PM seems to be something like that `data-feature="ipc.[close|open]"` but if it's demanded as it

Reply:
:+1: why  not, I was not aware of that convention